README.md

```diff
@@ -23,8 +23,8 @@ mkvirtualenv forest-service-prototype
 pip install -r requirements.txt
 createdb forest-service-prototype
 cd forestserviceprototype
-./manage.py migrate
-./manage.py runserver
+forestserviceprototype/manage.py migrate
```

Comment:
I don't think this is necessary because we are specifying that the cd into in the line 25.